clang/test/SemaSYCL/intel-max-global-work-dim-device.cpp

```diff
@@ -33,6 +33,20 @@ struct Func {
   [[intelfpga::max_global_work_dim(2)]] void operator()() const {}
 };
 
+//Checking of duplicate argument values
```

Comment:
Whitespace between `//` and the rest of the comment, and add a period to the end of the comment. Same below.